blocks/page_list/templates/itinerary/Header.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+export default class Header extends React.Component {
+  constructor(...args) {
+    super(...args);
+    this.state = {
+      editable: false,
+      newDescription: null,
+      newTitle: null,
+    }
+  }
+
+  getDefaultProps() {
```

Comment:
Don't use this, or 20, as methods with returns. These should be defined as static properties on the class (they also work on stateless component functions).
https://facebook.github.io/react/docs/reusable-components.html#es6-classes